packages/nocodb/src/lib/meta/api/metaDiffApis.ts

```diff
@@ -532,18 +539,229 @@ async function getMetaDiff(
 
 export async function metaDiff(req, res) {
   const project = await Project.getWithInfo(req.params.projectId);
-  const base = project.bases[0];
+  let changes = []
+  for (const base of project.bases) {
+    try {
+      // @ts-ignore
+      const sqlClient = NcConnectionMgrv2.getSqlClient(base);
+      changes = changes.concat(await getMetaDiff(sqlClient, project, base));
+    } catch (e) {
+      console.log(e);
+    }
+  }
+
+  res.json(changes);
+}
+
+export async function baseMetaDiff(req, res) {
+  const project = await Project.getWithInfo(req.params.projectId);
+  const base = await Base.get(req.params.baseId);
+  let changes = []
 
-  // @ts-ignore
   const sqlClient = NcConnectionMgrv2.getSqlClient(base);
-  const changes = await getMetaDiff(sqlClient, project, base);
+  changes = changes.concat(await getMetaDiff(sqlClient, project, base));
```

Comment:
Is `concat` even required here since `changes` is always an empty array at this point?